cid/helpers/athena.py

```diff
@@ -155,6 +159,32 @@ def WorkGroup(self, name: str):
         self._WorkGroup = name
         logger.info(f'Selected Athena WorkGroup: "{self._WorkGroup}"')
 
+    def _ensure_workgroup(self, name: str) -> str:
```

Comment:
I would suggest the we need to check first if workgroup exists, create if not 

 then if it has output bucket - propose a customer create or use existing? if choose use existing - provide the list of buckets in the current region